pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.554.3</version>
+        <version>1.596.1</version>
```

Comment:
Useful to clarify (in the PR, or in a comment in the POM itself) which core APIs you are using that requires this LTS version.